main/perl-http-body/APKBUILD

```diff
@@ -12,35 +12,28 @@ makedepends="perl-dev perl-test-deep"
 install=""
 subpackages="$pkgname-doc"
 source="http://search.cpan.org/CPAN/authors/id/G/GE/GETTY/HTTP-Body-$pkgver.tar.gz
-	CVE-2013-4407.patch
 	"
 
-_builddir="$srcdir"/HTTP-Body-$pkgver
+builddir="$srcdir"/HTTP-Body-$pkgver
+
+# secfixes:
+#   1.22-r0:
+#     - CVE-2013-4407
```

Comment:
This secfixes comment is not correct. It indicates that the versions prior to 1.22-r0 is vulnerable to CVE-2013-4407, which is not true. We had a patch for it, which is removed with this commit.

The secfix comment needs to say which .apk version we fixed this CVE in alpine.

Reply:
use git log to find out exactly when we fixed this issue.

Reply:
Thank you, I just corrected that line